.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: go
 
 go:
-  - '1.12'
   - '1.13'
-  - '1.14'
+  - '1.14.2'
```

Comment:
Current version is 1.14.4. We recently updated Prebid Cache to Go 1.14.2 without any production issues, so using the same version here for consistency. If you feel differently regarding the minor version, please let me know.